libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/MapboxNavigationTelemetry.kt

```diff
@@ -660,45 +751,75 @@ internal object MapboxNavigationTelemetry : MapboxNavigationTelemetryInterface {
+        navigationEvent.absoluteDistanceToDestination = obtainAbsoluteDistance(
+            callbackDispatcher.getLastLocation(),
+            obtainRouteDestination(directionsRoute)
+        )
         navigationEvent.percentTimeInPortrait = lifecycleMonitor?.obtainPortraitPercentage() ?: 100
-        navigationEvent.percentTimeInForeground = lifecycleMonitor?.obtainForegroundPercentage() ?: 100
+        navigationEvent.percentTimeInForeground =
+            lifecycleMonitor?.obtainForegroundPercentage() ?: 100
         navigationEvent.distanceCompleted = dynamicValues.distanceCompleted.get().toInt()
         navigationEvent.distanceRemaining = dynamicValues.distanceRemaining.get().toInt()
         navigationEvent.durationRemaining = dynamicValues.durationRemaining.get()
         navigationEvent.eventVersion = EVENT_VERSION
         navigationEvent.estimatedDistance = directionsRoute.distance()?.toInt() ?: 0
         navigationEvent.estimatedDuration = directionsRoute.duration()?.toInt() ?: 0
         navigationEvent.rerouteCount = dynamicValues.rerouteCount.get()
-        navigationEvent.originalEstimatedDistance = callbackDispatcher.getOriginalRouteReadOnly()?.route?.distance()?.toInt() ?: 0
-        navigationEvent.originalEstimatedDuration = callbackDispatcher.getOriginalRouteReadOnly()?.route?.duration()?.toInt() ?: 0
-        navigationEvent.stepCount = obtainStepCount(callbackDispatcher.getRouteProgress().routeProgress.route)
-        navigationEvent.originalStepCount = obtainStepCount(callbackDispatcher.getOriginalRouteReadOnly()?.route)
-        navigationEvent.legIndex = callbackDispatcher.getRouteProgress().routeProgress.route.routeIndex()?.toInt() ?: 0
-        navigationEvent.legCount = callbackDispatcher.getRouteProgress().routeProgress.route.legs()?.size ?: 0
-        navigationEvent.stepIndex = callbackDispatcher.getRouteProgress().routeProgress.currentLegProgress?.currentStepProgress?.stepIndex ?: 0
+        navigationEvent.originalEstimatedDistance =
+            callbackDispatcher.getOriginalRouteReadOnly()?.route?.distance()?.toInt() ?: 0
+        navigationEvent.originalEstimatedDuration =
+            callbackDispatcher.getOriginalRouteReadOnly()?.route?.duration()?.toInt() ?: 0
+        navigationEvent.stepCount =
+            obtainStepCount(callbackDispatcher.getRouteProgress().routeProgress.route)
+        navigationEvent.originalStepCount =
+            obtainStepCount(callbackDispatcher.getOriginalRouteReadOnly()?.route)
+        navigationEvent.legIndex =
+            callbackDispatcher.getRouteProgress().routeProgress.route.routeIndex()?.toInt() ?: 0
+        navigationEvent.legCount =
+            callbackDispatcher.getRouteProgress().routeProgress.route.legs()?.size ?: 0
+        navigationEvent.stepIndex =
+            callbackDispatcher
+            .getRouteProgress()
+            .routeProgress
+            .currentLegProgress
+            ?.currentStepProgress
+            ?.stepIndex
+            ?: 0
```

Comment:
Found some formatting mismatch between `ktlintFormat` and `Code | Reformat` `⌥⌘L` from the IDE that we should fix (I personally rely on `⌥⌘L` heavily 😅) 👀 

```kotlin
callbackDispatcher
.getRouteProgress()
.routeProgress
.currentLegProgress
?.currentStepProgress
?.stepIndex
?: 0
```

vs

```kotlin
callbackDispatcher
     .getRouteProgress()
    .routeProgress
    .currentLegProgress
    ?.currentStepProgress
    ?.stepIndex
    ?: 0
```

Same was noticed in the previous PR https://github.com/mapbox/mapbox-navigation-android/pull/3188#discussion_r442400616

Should we `Code | Reformat` `⌥⌘L` the whole project as part of this PR? It seems `ktlint` is not able to "fix" indentation.

The same thing happens to a bunch of other files.

Reply:
```
callbackDispatcher
    .getRouteProgress()
    .routeProgress
    .currentLegProgress
    ?.currentStepProgress
    ?.stepIndex
    ?: 0
```
this is actually right kotlin style([indent 4 spaces](https://developer.android.com/kotlin/style-guide#indentation)). looks like a gap at ktlint setup. I'll update guide 👍

Reply:
@Guardiola31337 it's a ktlint issue, let's work with current version and keep updating for new